src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = days * 40;
+  const firstNumberOfDays = 3;
+  const secondNumberOfDays = 7;
+  const firstDiscount = 20;
+  const secondDiscount = 50;
```

Comment:
```suggestion
  const shortTerm = 3;
  const longTerm = 7;
  const smallDiscount = 20;
  const largeDiscount = 50;
```
Так більш зрозуміліше